cfn-resources/x509-authentication-database-user/README.md

```diff
@@ -0,0 +1,59 @@
+# MongoDb::Atlas::X509AuthenticationDatabaseUser
+## Description
+Returns, edits, and removes user-managed X.509 configurations.
+Also returns and generates MongoDB Cloud-managed X.509 certificates for database users.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Local Testing
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/x509-authentication-database-user
+./test/x509authenticationdatabaseuser.create-sample-cfn-request.sh YourProjectID > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
+```
+
```

Comment:
Missing link to example folder, this will make it easier for users to find file that you created here: examples/x509-authentication-db-user/x509selfmanaged.json